lib/source/search/index.html.erb

```diff
@@ -0,0 +1,26 @@
+---
+title: Search
+index: false
+prevent_indexing: true
+hide_in_navigation: true
+---
+
+<div id="search-help" class="search-help">
+  <%# strange id is to avoid clash with search input id #%>
+  <h1 id="how-to-search">Search</h1>
+
+  <p class="govuk-body search-help__no-js">
+    This page requires JavaScript to perform searches. Please enable JavaScript, or use the search input to search using Google.
+  </p>
+
+  <p class="govuk-body">
```

Comment:
If I understand right, I think we can remove this paragraph? Seems to duplicate the one above.

Reply:
This paragraph currently appears whether or not the user has JavaScript enabled; I guess we could make it hidden until some JS unhides it?

Reply:
Do you know what triggers it to appear if you have JS on? I could only get it to show up if I switched JS off.
(Btw if this paragraph was already part of the search before this PR, please leave as is - don't want to inadvertently increase the scope of the PR)

Reply:
Ah, sorry, I was confused, I thought your original comment was referring to the paragraph reading 'Search the documentation using the search input.'.

Reply:
I'm not sure what you mean by this paragraph duplicating the one above.

Reply:
Ah yes it was! I just can't get that second paragraph to show up, unless I turn JS off. So wondered in what circumstances it shows up.

Reply:
Can you see it if you visit https://deploy-preview-147--govuk-frontend-docs-preview.netlify.app/search/?

It appears when the user clicks the search button but has not entered a search query.

Reply:
Ah yes I can see it! In that case we should probably tweak this a bit to be clearer. So I'd suggest replacing "Search the documentation using the search input." with "Enter a search term in the search box, then select **Search**".

Reply:
I'm not sure about 'select **Search**', as the search button doesn't actually say search on it... what about 'click the search button', or something similar?

Reply:
Yep that works! Just use 'select' instead of 'click' as it's more accessible: https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#click

Reply:
Ah cool, didn't know that!